locales/id-ID/translation.json

```diff
@@ -427,25 +427,25 @@
   "welcomeScreenPassphraseMessage": "Jika Anda ingin menggunakan <1>frasa sandi</1>, kunci dompet yang baru Anda buat.",
   "with": "dengan",
-  "You are currently connected to the {{ currentNetwork }} network. Make sure to connect to the testnet network to see your tokens.": "You are currently connected to the {{ currentNetwork }} network. Make sure to connect to the testnet network to see your tokens.",
+  "You are currently connected to the {{ currentNetwork }} network. Make sure to connect to the testnet network to see your tokens.": "Anda saat ini terhubung ke jaringan {{ currentNetwork }}. Pastikan untuk terhubung ke jaringan testnet untuk melihat token Anda.",
   "You can disable this confirmation step from the wallet settings.": "Anda dapat menonaktifkan langkah konfirmasi ini dari pengaturan dompet.",
   "You can download the address transaction history for a selected time period. This can be useful for tax reporting.": "Anda dapat mengunduh riwayat transaksi alamat untuk jangka waktu yang dipilih. Ini dapat berguna untuk pelaporan pajak.",
   "You have great ideas you want to share?": "Anda memiliki ide hebat yang ingin Anda bagikan?",
   "You have questions about transfers ? Click here!": "Anda memiliki pertanyaan tentang transfer? Klik disini!",
-  "You have reached the maximum calls limit. Please try again in a few minutes.": "You have reached the maximum calls limit. Please try again in a few minutes.",
+  "You have reached the maximum calls limit. Please try again in a few minutes.": "Anda telah mencapai batas panggilan maksimum. Silakan coba lagi dalam beberapa menit nanti.",
   "You will not be able to recover your account with the private key!": "Anda tidak akan dapat memulihkan akun Anda dengan kunci pribadi!",
   "Your address": "Alamat Anda",
   "Your addresses": "Alamat Milik Anda",
   "Your CSV file has been generated successfully.": "File CSV Anda telah berhasil dibuat.",
   "Your CSV file is being compiled in the background.": "File CSV Anda sedang dikompilasi di latar belakang.",
   "Your Wallet": "Dompet Anda",
-  "Additionally, {{ amount }} ALPH needs to be sent. Click here for more info.": "Additionally, {{ amount }} ALPH needs to be sent. Click here for more info.",
-  "Address unknown tokens": "Address unknown tokens",
-  "Experimental": "Experimental",
-  "Public key": "Public key",
-  "Private key": "Private key",
-  "Address keys export": "Address keys export",
-  "Public key copied.": "Public key copied.",
-  "Could not copy public key.": "Could not copy public key.",
-  "Copy the keys of an address.": "Copy the keys of an address."
+  "Additionally, {{ amount }} ALPH needs to be sent. Click here for more info.": "Selain itu, {{ amount }} ALPH diperlukan untuk dikirim. Klik di sini untuk info lebih lanjut.",
+  "Address unknown tokens": "Alamat token yang tidak diketahui",
```

Comment:
Does tihs mean "Address unknown tokens"? Because Deepl tells me that it translates to "Unknown token address". It's not the address that is unknown, but the tokens.

Reply:
@ykris45 how Deepl translates to "Unrecognized token address". It stills conveys the message that the address is unrecognised, whereas it is the tokens that are unrecognized. Not sure how it works in indonesian, I just wanna make sure we are not making a mistake! Could you please find someone who speaks Indonesian to verify this?